pages/download.py

```diff
@@ -111,16 +111,14 @@ def get_content(self):
 """)))+div("macos",md(_("""
 ### OS X
 
-[Install Freenet 0.7](assets/jnlp/freenet.jnlp) using Java Web Start.  
-This requires that Java is installed and that Java Web Start is enabled. If
-it doesn't work, try [installing Java](http://www.java.com/),
-then downloading [the installer](assets/jnlp/freenet_installer.jar) and
-opening it. It might take a moment to launch.
-
-**Note:** Mavericks does not ship with Java Web Start enabled. We would like
-to make a _.dmg_ for easier OS X installation but don't know how yet. If you
-are a developer and would like to join us and help it would be much
-appreciated!
+Download and run [the installer](https://github.com/steveatinfincia/mactray/releases/download/v2.0.0/FreenetTray_2.0.0-signed.zip)
+
+It will automatically install Freenet and other required components for you.
+When done, your default browser will automatically open up to Freenet's
```

Comment:
He was testing on a machine with Freenet already installed, in that case the app automatically found his installation and silently put itself in the tray (which is something I'm considering changing). During installation though the very last step is for the user to click continue which does open the web interface.